libindy/include/indy_ledger.h

```diff
@@ -68,6 +68,39 @@ extern "C" {
                                                                      const char*     request_result_json)
                                                );
 
+
+    /// Signs request message.
+    ///
+    /// Adds submitter information to passed request json, signs it with submitter
+    /// sign key (see wallet_sign).
+    ///
+    /// #Params
+    /// command_handle: command handle to map callback to caller context.
+    /// wallet_handle: wallet handle (created by open_wallet).
+    /// submitter_did: Id of Identity stored in secured Wallet.
+    /// request_json: Request data json.
+    /// cb: Callback that takes command result as parameter.
+    ///
+    /// #Returns
+    /// Signed request json.
+    ///
+    /// #Errors
+    /// Common*
+    /// Wallet*
+    /// Ledger*
+    /// Crypto*
+
+    extern indy_error_t indy_sign_and_submit_request(indy_handle_t command_handle,
+                                                         indy_handle_t wallet_handle,
+                                                         const char *    submitter_did,
+                                                         const char *    request_json,
+
+                                                         void           (*cb)(indy_handle_t xcommand_handle,
+                                                                              indy_error_t  err,
+                                                                              const char*     signed_request_json)
+                                                         );
+
+
```

Comment:
No, I don't see a use case for this request. We sign request to be only verified by pool nodes.